.github/workflows/test.yml

```diff
@@ -58,6 +58,16 @@ jobs:
         run: |
           sudo apt-get install -y expect
 
+      - name: Install homebrew package manager
```

Comment:
Do you need to reinstall brew? https://github.com/actions/runner-images/blob/macos-14-arm64/20240422.3/images/macos/macos-14-arm64-Readme.md#package-management says that brew is present on the image.

If brew was not already present, this step would be incomplete since it does not add brew to the path for the following step.

`eval "$(/opt/homebrew/bin/brew shellenv)"`